code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -436,10 +444,17 @@ func (fsh *StorageHandler) CommitWrite(ctx context.Context, r *http.Request) (*b
 		return nil, err
 	}
 
+	elapsedWritePreRedeem := time.Since(startTime) - elapsedAllocation - elapsedGetLock -
+		elapsedGetConnObj - elapsedVerifyWM
+
 	err = connectionObj.ApplyChanges(ctx, writeMarker.AllocationRoot, writeMarker.Timestamp)
 	if err != nil {
 		return nil, err
 	}
+
+	elapsedApplyChanges := time.Since(startTime) - elapsedAllocation - elapsedGetLock -
+		elapsedGetConnObj - elapsedVerifyWM - elapsedWritePreRedeem
```

Comment:
Re-declaring the startTime or a new timestamp will make the elapsed* calculation much simpler. So the elapsedApplyChanges would be simple be `time.Since(startTime)`.  So as all the elapsed* above.

Reply:
This will be deleted once we are over with loadtest.
However I will make the change for simplicity